src/mlpack/tests/lin_alg_test.cpp

```diff
@@ -52,10 +50,13 @@ BOOST_AUTO_TEST_CASE(TestCenterA)
   //  [-8 -4  0  4  8]]
   for (int row = 0; row < 5; row++)
     for (int col = 0; col < 5; col++)
-      BOOST_REQUIRE_CLOSE(tmp_out(row, col), (double) (col - 2) * row, 1e-5);
+    {
+      REQUIRE(tmp_out(row, col) ==
+          Approx((double) (col - 2) * row).epsilon(1e-7));
+    }
```

Comment:
Just another simple style comment; if the inner loop now has braces, can you also add braces to the outer loop?  (This is mentioned somewhere in the style guide.)